_includes/block/head.html

```diff
@@ -23,12 +23,29 @@
     <link rel="canonical" href="{{ site.url }}{{ page.url}}">
   {% endif %}
 
-  <meta name="og:image" content="{{ site.url }}/assets/images/banner.png">
+  <meta property="og:locale" content="en_US">
+  <meta property="og:site_name" content="{{ site.title }}">
+  <meta property="og:url" content="{{ site.url }}{{ page.url}}">
+
+  {% if page.cover %}
+    <meta name="og:image" content="{{ site.url }}{{ page.cover }}">
```

Comment:
Can you update these to use the `property` attribute instead of `name`? I've been seeing these warnings in the [Facebook debugger](https://developers.facebook.com/tools/debug/) for the live site:

<img width="1001" alt="screen shot 2018-09-12 at 8 23 15 am" src="https://user-images.githubusercontent.com/1934719/45435357-462b6000-b665-11e8-8257-dca81cfc5e0d.png">